starwars/src/App.js

```diff
@@ -27,6 +28,11 @@ class App extends Component {
     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
+        <div>
+          {this.state.starwarsChars.map((character, i) => {
+            return <Character key={i} character={character} />
```

Comment:
Nicely done here. Using a reusable component that is being mapped over and given a key!